tools/capacity-planner-sheets-extension/sheets.js

```diff
@@ -0,0 +1,185 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * 
+ * 
+ * Convenience functions for creating, writing to, and formatting sheets.
+ */
+
+BACKGROUND_COLORS = ["#ead1dc", "#cfe2f3", "#d0e0e3", "#fff2cc", "#f4cccc"];
+
+/**
+ * Creates a new sheet with the given name.
+ * NOTE: If a sheet with sheetName already exists, this function will delete it and create a new
+ * sheet with the same name.
+ * @param {string} The name of the new sheet
+ */
+function createSheet_(sheetName){
+  let ss = SpreadsheetApp.getActive();
```

Comment:
For variables who's values won't be re-assigned consider using `const` instead of `let`. When using const, the value it holds *can still be mutable* e.g. `ss.deleteSheet(sheet);` but the identifier cannot be re-assigned. I would suggest using it accordingly where you see that fit.

Reply:
Thanks for the explanation! I've removed all instances of `var` and switched to `let` and `const.